conans/client/installer.py

```diff
@@ -329,6 +320,25 @@ def _handle_node_editable(self, node):
                                        settings=node.conanfile.settings,
                                        options=node.conanfile.options)
 
+            build_folder = editable_cpp_info.folder(node.ref, "build",
+                                                    settings=node.conanfile.settings,
+                                                    options=node.conanfile.options)
```

Comment:
`editable_cpp_info` object is created from a `package_layout`, so it shouldn't accept a `ref` in its functions, it must be already coupled with a reference.

Reply:
I didn't put this here, it is just copied form the above ``editable_cpp_info.apply_to(node.ref,``
I agree this should be fixed, but that is another PR, this is focused on the Workspaces

Reply:
Didn't realize about the previous, I thought it was the first time, ok, you win 😆